packages/next-swc/crates/napi/src/minify.rs

```diff
@@ -109,6 +112,10 @@ fn patch_opts(opts: &mut JsMinifyOptions) {
         inline: Some(TerserInlineOption::Num(2)),
         ..Default::default()
     });
+    opts.mangle = BoolOrDataConfig::from_obj(MangleOptions {
+        reserved: vec!["AbortSignal".into()],
```

Comment:
Should we also put it in the reserved list for `terser` options?

Reply:
I think so, will do